src/icons/icons.qrc

```diff
@@ -267,7 +267,9 @@
         <file>qbt-theme/document-properties.svg</file>
         <file>qbt-theme/document-save.svg</file>
         <file>qbt-theme/download.svg</file>
-        <file>qbt-theme/edit-clear-history.svg</file>
+        <file>qbt-theme/drop-down.svg</file>
+        <file>qbt-theme/drop-up.svg</file>
+				<file>qbt-theme/edit-clear-history.svg</file>
```

Comment:
Wrong indentation.

Reply:
sorry, but i didn't get it, do you mean by arranging them alphabetically?

Reply:
Xd it looks fine on my desktop github.